somleng_scfm_tools/upload_csv_contacts_to_scfm.py

```diff
@@ -0,0 +1,63 @@
+import json
+import csv
+import argparse
+import http.client
+
+from core_data_modules.logging import Logger
+
+log = Logger(__name__)
+log.set_project_name("UploadCsvContactsToScfm")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Uploads a phone number:metadata from a csv file with header "
```

Comment:
I don't understand this description. It reads as if it's going to upload one column of a CSV, the column with the header "phone number:metadata"